starter/cmd/startercli/locales/en/completion.en.yaml

```diff
@@ -31,11 +31,11 @@ completion.bash.cmd.longDescription:
   one: '''
   Generate command completion script for Bash shell.
   Installing on Linux:
-  1. Run `cli completion bash > cli_completions` to create the script file.
-  2. Run `sudo mv cli_completions /etc/bash_completion.d/cli` to place the script in a special Bash completions directory.
+  1. Run `startercli completion bash > startercli_completions` to create the script file.
```

Comment:
Maybe ignore cli name here.
Also CLI name can be build var

Reply:
Resolved via commit: https://github.com/aerogear/charmil/pull/243/commits/d9daf62f3f2be1b9de2af2fc23a9a0814258e776